fixtures/worker-logs/wrangler.module.jsonc

```diff
@@ -1,5 +1,6 @@
 {
 	"name": "worker-logs",
-	"compatibility_date": "2022-03-31",
+	"compatibility_date": "2025-09-01",
```

Comment:
probably not necessary

Reply:
probably not, can you revert and check?

Reply:
it is necessary to avoid using nodejs_compat_v1